### PR TITLE
Fix my_byteorder.h related build error

### PR DESCRIPTION
**Pull request by kenhys**

### CODE REVIEW

File: lib/mrn_value_decoder.hpp [hunk 1]
```diff
@@ -21,6 +21,7 @@
 #define MRN_VALUE_DECODER_HPP_
 
 #include <mrn_mysql.h>
+#include <mrn_mysql_compat.h>
```
Comment (kou) [inline lib/mrn_value_decoder.hpp]: Can we move to this `.cpp`?

File: lib/mrn_value_decoder.cpp [hunk 2]
```diff
@@ -17,9 +17,10 @@
   Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
 */
 
+#include <mrn_mysql_compat.h>
```
Comment (kou) [inline lib/mrn_value_decoder.cpp]: Including after `mrn_value_decoder.hpp` is better to not affect definitions in `mrn_value_decoder.hpp`.

Comment (kenhys) [reply lib/mrn_value_decoder.cpp]: I got it.

**Comment by kenhys** [comment]
@kou could you review again, please?

**Pull request merged**